src/app/dashboard/competitions/[competitionId]/register-form/create/page.tsx

```diff
@@ -0,0 +1,27 @@
+import { notFound } from "next/navigation";
+
+import { getCompetitionById } from "@/lib/db/competition";
+
+import CreateCompetitionRegisterForm from "./components/CreateCompetitionRegisterForm";
+
+interface CreateRegisterFormPageProps {
```

Comment:
```suggestion
interface CreateCompetitionRegisterFormPageProps {
```

Para mantener consistencia con el nombre del componente